integration-tests/15_achievements.ts

```diff
@@ -221,12 +216,15 @@ void test('Reward pupil conducted match appointment', async () => {
         }
     `);
     const [match] = matches;
+
+    const dates = createDates();
+    const appointments = await generateLectures(dates, match, student.userID, pupil.userID);
     await client.request(`
```

Comment:
(funny that previously one could track meeting joins for non existing appointments ...)